internal/gui/keybindings.go

```diff
@@ -8,15 +8,26 @@ import (
 )
 
 var inspectorFormat = map[string]string{
-	`trips`    : "Date: %s\nCave: %s\nCavers: %s\nNotes: %s",
-	`people`   : "Name: %s\nClub: %s\nCount: %d",
-	`locations`: "Name: %s\nRegion: %s\nCountry: %s\nSRT: %v\nVisits: %d",
+	`trips` : "Date: %s\nCave: %s\nCavers: %s\nNotes: %s",
+	`cavers`: "Name: %s\nClub: %s\nCount: %d",
+	`caves` : "Name: %s\nRegion: %s\nCountry: %s\nSRT: %v\nVisits: %d",
 }
 
 func (g *Gui) setGlobalKeybinding(event *tcell.EventKey) {
+	/*switch event.Key() {
```

Comment:
Remove or enable changing of pages using <kbd>Tab</kbd> and <kbd>Shift</kbd>+<kbd>Tab</kbd>

Reply:
This will likely be a larger issue than this PR. So it might be best for the moment to allow the code to stay albeit commented out.